### PR TITLE
{lib}[GCC/8.3.0] Brotli v1.0.9

### PR DESCRIPTION
**Pull request by fizwit**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Research Center - Seattle Washington - US
+#
+easyblock = 'CMakeMake'
+name = 'brotli'
```
Comment (Micket) [inline easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb:8]: I think we should go for `Brotli` as the name here

**Review by Micket — DISMISSED** [review]

File: easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb (hunk 1 above)
Comment (fizwit) [reply easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb:8]: Brotli-1.0.9-GCC-8.3.0.eb

File: easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+version = '1.0.9'
+
+homepage = 'https://github.com/google/brotli'
+description = """Brotli is a generic-purpose lossless compression algorithm that compresses data using a combination
+ of a modern variant of the LZ77 algorithm, Huffman coding and 2nd order context modeling, with a compression ratio
+ comparable to the best currently available general-purpose compression methods. It is similar in speed with deflate
+ but offers more dense compression.
+The specification of the Brotli Compressed Data Format is defined in RFC 7932."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
```
Comment (smoors) [inline easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb:18]: @fizwit other compression software are built with `GCCcore` toolchain, I think we should go with that here as well?

File: easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
```
Comment (smoors) [inline easybuild/easyconfigs/b/brotli/brotli-1.0.9-GCC-8.3.0.eb:1]: please remove this stale easyconfig?

**Review by smoors — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb (hunk 2 above)
Comment (fizwit) [reply easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb:18]: Brotli was required by some Python package, required by some bioinformatics thing. I'm so deep into the dependencies I forgot what it was.

**Review by fizwit — COMMENTED** [review]
removed stale version of Brotili

File: easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb (hunk 2 above)
Comment (smoors) [reply easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb:18]: but does not matter, right? if the GCC-based version can be used as dep, GCCcore-based should work too.

File: easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# easybuild easyconfig
```
Comment (smoors) [inline easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb:1]: @fizwit this easyconfig should also be removed

Comment (fizwit) [reply easybuild/easyconfigs/b/Brotli/Brotli-1.0.9-GCC-8.3.0.eb]: removing Brotli-1.0.9-GCC-8.3.0.eb

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node373.hydra.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz (skylake_avx512), Python 2.7.5
See https://gist.github.com/5d846cedcbf93fe22f1fb6704e81f265 for a full test report.

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @fizwit!

**Pull request merged**